packages/apps/desktop-viewer-test/package.json

```diff
@@ -92,12 +92,11 @@
     "cpx2": "4.2.0",
     "cross-env": "^5.2.1",
     "electron-devtools-installer": "^2.2.4",
-    "eslint": "^7.32.0",
     "npm-run-all": "^4.1.5",
     "rimraf": "^3.0.2",
     "sass": "^1.64.2",
     "typescript": "~5.0.4",
-    "webpack": "4.42.0"
+    "webpack": "^5.1.2"
```

Comment:
why did we jump majors here?
did we react to the breaking changes? this "instance" of webpack is being used to bundle the backend